tests/parser/parser.cpp

```diff
@@ -1446,11 +1446,12 @@ TEST(Parser, invalidModelWithAllCausesOfIssues)
         case (libcellml::Issue::Cause::VARIABLE):
             foundCause.at(6) = true;
             break;
+        case libcellml::Issue::Cause::ANALYSER:
+        case libcellml::Issue::Cause::GENERATOR:
         case libcellml::Issue::Cause::MATHML:
```

Comment:
See earlier comment.  Are causes entities, or could they be processing classes too?

Reply:
Processing classes? Anyway, for another issue, methinks.

Reply:
Urgh.  I think it should just go.  The Analyser and Generator have (or should have) their own Loggers, so this just makes no sense ...

Reply:
Gone as part of commit 395a18e.

About `Generator`, it doesn't need to inherit from `Logger` in fact, since it doesn't generate issues anymore. So, I have removed it in commit 577e1af1.